.redhat-ci.yml

```diff
@@ -13,11 +13,12 @@ context: fedora/25/atomic
 required: true
 
 env:
-  DOCKER_STORAGE_SETUP: ./docker-storage-setup.sh
 
 tests:
   - systemctl stop docker
   - rm -rf /var/lib/docker
+  - ostree admin unlock
+  - make install
```

Comment:
I'm surprised this works. Didn't realize `make` was in Fedora AH. The way this workflow is done usually is to `make install DESTDIR=/host` from inside a build container. But since you don't actually need to compile anything and `make` is available... why not.